app/upgrades_test.go

```diff
@@ -496,6 +496,18 @@ func (s *UpgradeTestSuite) TestTourmaline() {
 	s.AssertUpgradeHandlerLogs("tourmaline", expInLog, nil)
 }
 
+func (s *UpgradeTestSuite) TestUmberRC1() {
+	expInLog := []string{}
```

Comment:
```suggestion
	expInLog := []string{
		"INF Starting module migrations. This may take a significant amount of time to complete. Do not restart node.",
		"INF removing all delegations from validators that have been inactive (unbonded) for 21 days",
	}
```

Same goes for `TestUmber` below.